### ISSUE TITLE
Error: The browser/environment lacks native support for critical functionality used by the PDF.js library (e.g. `ReadableStream` and/or `Promise.allSettled`); please use an ES5-compatible build instead.

### ISSUE THREAD

**Issue by ttys3** [opendocsg/pdf2md#45]
run `npx @opendocsg/pdf2md --inputFolderPath=[your input folder path] --outputFolderPath=[your output folder path] --recursive`

got 

```
Error: The browser/environment lacks native support for critical functionality used by the PDF.js library (e.g. `ReadableStream` and/or `Promise.allSettled`); please use an ES5-compatible build instead.
```

**Comment by ttys3** [comment]
solution https://stackoverflow.com/a/64189798/13267147

**Comment by LoneRifle** [comment]
Thanks for filing this! We're all pretty busy at this time, so it might take a while for us to get to this issue. If you want to file a PR, I'll be happy to review!

**Comment by LoneRifle** [comment]
Should be fixed in 0.1.21, please reopen if otherwise!
